docs/details/dashboards/dashboard-inventory.md

```diff
@@ -34,6 +40,18 @@ Each `node_id` is associated with a `machine_id` (from `/etc/machine-id`). Nodes
 
 By expanding the entry from the options column, you can check the node labels and attributes.
 
+Starting with PMM 2.38.0, you can see the number of agents running on any particular node. When you click on any node, the UI navigates to the view of agents, which is filtered to display only agents related to that specific node. 
+
+To see the details of the agents running, do the following:
+
+1.On the **Nodes** tab,  under the **Monitoring** column, click **OK** or **Failed** depending on the status of the node that you have selected. A page that provides the user with crucial information regarding the total number of agents deployed on that node is displayed.. A page that provides the user with crucial information regarding the total number of agents deployed on that node is displayed.
```

Comment:
Sentence is duplicated:
A page that provides the user with crucial information regarding the total number of agents deployed on that node is displayed.